pkg/cmd/server/start/start_node.go

```diff
@@ -260,7 +260,7 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 	if err != nil {
 		return err
 	}
-	glog.Infof("Starting OpenShift node %s (%s)", config.KubeletServer.HostnameOverride, version.Get().String())
+	glog.Infof("Starting node %s (%s)", config.KubeletServer.HostnameOverride, version.Get().String())
```

Comment:
Any reason we say "Starting a node" in one message and "Starting node" in this one